interfaces/builtin/intel_mei.go

```diff
@@ -0,0 +1,50 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2019 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+const intelMEISummary = `allows access to the intel MEI management interface`
+
+const intelMEIBaseDeclarationSlots = `
+  intel-mei:
+    allow-installation:
+      slot-snap-type:
+        - core
+    deny-auto-connection: true
+`
+
+const intelMEIConnectedPlugAppArmor = `
+# Description: Allow access to the Intel MEI management interface.
+/dev/mei[0-9]+ rw,
+`
+
+var intelMEIConnectedPlugUDev = []string{`KERNEL=="mei[0-9]+"`}
```

Comment:
I have a /dev/mei0 available and udevadm tells me:
```
$ udevadm info /dev/mei0
P: /devices/pci0000:00/0000:00:16.0/mei/mei0
N: mei0
E: DEVNAME=/dev/mei0
E: DEVPATH=/devices/pci0000:00/0000:00:16.0/mei/mei0
E: MAJOR=239
E: MINOR=0
E: SUBSYSTEM=mei
```

I tried to get it to udev tag locally but it didn't work. I think we need to do this (which worked locally):
```
[]string{`SUBSYSTEM=="mei"`}
```